index.html

```diff
@@ -15,7 +15,7 @@ <h2>MiniFluid</h2>
 #  */p=a;              e,p&lt;r;                                        p+=5)//  #
 #  define              z(e,i)                                        f(p,p/*  #
 ## */[i]=e)f(q,w=cabs  (d=*p-  *q)/2-     1)if(0  &lt;(x=1-      w))p[i]+=w*/// ##
-   double complex a [  97687]  ,*p,*q     ,*r=a,  w=0,d;    int x,y;char b/* ##
+## double complex a [  97687]  ,*p,*q     ,*r=a,  w=0,d;    int x,y;char b/* ##
```

Comment:
this "  " was present in the original C file. I guess we can keep it until we do a JS quine ourselves :p